src/components/VideoPlayerContexts/VideoPopoverMenuContext.tsx

```diff
@@ -46,37 +51,30 @@ function VideoPopoverMenuContextProvider({children}) {
         items.push({
             icon: Expensicons.Meter,
             text: translate('videoPlayer.playbackSpeed'),
-            subMenuItems: [
-                ..._.map(CONST.VIDEO_PLAYER.PLAYBACK_SPEEDS, (speed) => ({
-                    icon: currentPlaybackSpeed === speed ? Expensicons.Checkmark : null,
-                    text: speed.toString(),
-                    onSelected: () => {
-                        updatePlaybackSpeed(speed);
-                    },
-                    shouldPutLeftPaddingWhenNoIcon: true,
-                })),
-            ],
+            subMenuItems: CONST.VIDEO_PLAYER.PLAYBACK_SPEEDS.map((speed) => ({
+                icon: currentPlaybackSpeed === speed ? Expensicons.Checkmark : null,
+                text: speed.toString(),
+                onSelected: () => {
+                    updatePlaybackSpeed(speed);
+                },
+                shouldPutLeftPaddingWhenNoIcon: true,
+            })),
         });
-
         return items;
     }, [currentPlaybackSpeed, downloadAttachment, translate, updatePlaybackSpeed, isOffline]);
 
     const contextValue = useMemo(() => ({menuItems, updatePlaybackSpeed}), [menuItems, updatePlaybackSpeed]);
-    return <VideoPopoverMenuContext.Provider value={contextValue}>{children}</VideoPopoverMenuContext.Provider>;
+    return <Context.Provider value={contextValue}>{children}</Context.Provider>;
 }
 
 function useVideoPopoverMenuContext() {
-    const context = useContext(VideoPopoverMenuContext);
-    if (context === undefined) {
+    const videoPopooverMenuContext = useContext(Context);
+    if (!videoPopooverMenuContext) {
```

Comment:
Same, please make sure that there is no side effects due to updated condition, since before the check was for undefined only, now it will cover null value as well.